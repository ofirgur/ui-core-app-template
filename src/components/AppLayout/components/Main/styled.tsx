import styled from '@emotion/styled';
import { grey } from '@mui/material/colors';

export const MainStyled = styled.main`
  flex: 1 0 auto;
  background: ${grey[100]};
`;
