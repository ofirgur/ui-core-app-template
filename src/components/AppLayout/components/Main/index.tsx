import React from 'react';
import Header from '../Header';
import Pages from '../Pages';
import { MainStyled } from './styled';

const Main = () => {
  return (
    <MainStyled>
      <Header />
      <Pages />
    </MainStyled>
  );
};

export default Main;
