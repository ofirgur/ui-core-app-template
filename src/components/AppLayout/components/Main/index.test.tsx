import * as React from 'react';
import { render } from '@testing-library/react';
import Main from '.';

describe('Main', () => {
  it('should render correctly', () => {
    const { container } = render(<Main />);
    expect(container).toMatchSnapshot();
  });
});
