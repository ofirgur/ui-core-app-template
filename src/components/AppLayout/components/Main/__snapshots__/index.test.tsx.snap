// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should render correctly 1`] = `
<div>
  <main
    class="sc-dkzDqf cFJZcl"
  >
    <h2
      class="sc-bczRLJ fqvweB"
    >
      Header
    </h2>
    <div
      class="sc-gsnTZi jpJQfJ"
    />
  </main>
</div>
`;
